versioned_docs/version-8.0/self-managed/zeebe-deployment/operations/health.md

```diff
@@ -4,60 +4,6 @@ title: "Health status"
 description: "This document analyzes health status checks and responses."
 ---
 
-## Broker
```

Comment:
🔧   If I'm not mistaken in 8.0 this is actually valid? can you double-check? It seems to have been removed with 8.1

Reply:
I went to check to query release-8-0-x benchmark and also does not work there this health status check to the broker.

The changes to the code were applied in this [PR](https://github.com/camunda/zeebe/pull/7984) on Oct13 2021 with no backports, How would I check to wich release this belongs to? 🤔